tests/benchmarks/test_fs.py

```diff
@@ -1,15 +1,18 @@
 import errno
-import pytest
 import shutil
+
+import pytest
 from reflink import reflink as pyreflink
 from reflink.error import ReflinkImpossibleError
-from dvc_objects.fs.system import reflink, hardlink, symlink
+
+from dvc_objects.fs.system import hardlink, reflink, symlink
 
 NLINKS = 10000
 
 
 @pytest.mark.parametrize(
-    "link", [pytest.param(pyreflink, id="pyreflink"), reflink, hardlink, symlink]
+    "link",
+    [pytest.param(pyreflink, id="pyreflink"), reflink, hardlink, symlink],
 )
```

Comment:
Do you have some ruff rules enabled locally that are not enabled in the repo?

Reply:
`ruff-format` on pre-commit changes this for me automatically. I don't have any other rules enabled.

Reply:
Doesn't seem like linter was failing before, so something is missing.